react/features/overlay/components/PageReloadOverlay.js

```diff
@@ -0,0 +1,298 @@
+/* global APP, AJS */
+
+import React, { Component } from 'react';
+
+import { randomInt } from '../../base/util/randomUtil';
```

Comment:
The exporting/importing of the `./react` features is package-based rather than file-based.